examples/app/duffing_oscillator.py

```diff
@@ -0,0 +1,125 @@
+''' Simulation of Duffing oscillator which is harmonic motion with a
+sinusoidal driving force and damping. It exhibits chaotic behaviour for some
+combinations of driving and damping parameters. This example demonstrates the
+use of mathtext on ``Div``, ``Paragraph`` and ``Slider`` objects, as well as
+axis labels, and also streaming data via a ``ColumnDataSource``.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.Figure.circle, bokeh.plotting.Figure.line, bokeh.models.sources.ColumnDataSource.stream,
+        bokeh.models.Div, bokeh.models.Paragraph, bokeh.models.Slider
+    :refs: :ref:`userguide_styling` > :ref:`userguide_styling_math`
+    :keywords: mathtext, latex
+'''
```

Comment:
nit, but I think all other examples docstrings always end with blank link 
```suggestion
    :keywords: mathtext, latex

'''
```

Reply:
Fixed it.